bridge-sdk/connectors/eth-connector/src/eth_connector.rs

```diff
@@ -97,10 +108,13 @@ impl EthConnector {
         )
         .await?;
 
+        tracing::info!(tx_hash = format!("{:?}", tx_hash), "Sent finalize deposit transaction");
```

Comment:
Why here it is tx_hash, but in all other places it is transaction_id?

I think better use tx_hash in all places

Reply:
Fixed